src/goto-instrument/count_eloc.cpp

```diff
@@ -141,3 +145,50 @@ void print_path_lengths(const goto_modelt &goto_model)
       ++n_reachable;
   std::cout << "Reachable instructions: " << n_reachable << "\n";
 }
+
+void print_global_state_size(const goto_modelt &goto_model)
+{
+  const namespacet ns(goto_model.symbol_table);
+
+  // if we have a linked object, only account for those that are used
+  // (slice-global-inits may have been used to avoid unnecessary initialization)
+  goto_functionst::function_mapt::const_iterator f_it =
+    goto_model.goto_functions.function_map.find(INITIALIZE_FUNCTION);
+  const bool has_initialize =
+    f_it != goto_model.goto_functions.function_map.end();
+  std::unordered_set<irep_idt> initialized;
+
+  if(has_initialize)
+  {
+    for(const auto &ins : f_it->second.body.instructions)
+    {
+      if(ins.is_assign())
+      {
+        const code_assignt &code_assign = to_code_assign(ins.code);
+        const symbol_exprt &symbol_expr = to_symbol_expr(code_assign.lhs());
```

Comment:
The LHS of an assign statement isn't always a symbol in __CPROVER_initialize -- for example, Java code will produce code such as `global_init->fieldname = NONDET(int);`.

Reply:
To that end you should probably add a couple of regression tests, or this will be vulnerable to moulding

Reply:
Both of the above comments are finally addressed.